.circleci/continue_config.yml

```diff
@@ -224,41 +221,39 @@ workflows:
   ci:
     jobs:
       - ci-base:
-          name: ci-base-<< matrix.python-version >>-<< matrix.extras >>
+          name: ci-base-<< matrix.python-version >><< matrix.extras >>
           matrix:
             parameters:
               python-version: [ "3.7", "3.8", "3.9", "3.10", "3.11" ]
-              extras: ["", "metrics"]
+              extras: ["", "-metrics"]
       - unit-test:
-          name: unit-test-<< matrix.python-version >>-<< matrix.extras >>
+          name: unit-test-<< matrix.python-version >><< matrix.extras >>
           matrix:
             parameters:
               python-version: [ "3.7", "3.8", "3.9", "3.10", "3.11" ]
-              extras: ["", "metrics"]
+              extras: ["", "-metrics"]
```

Comment:
In Mark's original PR, I asked him to remove the `-`, so that later handling of values, we don't have to think about it. Potentially we can use the value in `poetry install -- extras <extras>`. To me, the display of naming is less important than intuitive values.